src/jit/codegencommon.cpp

```diff
@@ -3012,9 +3012,8 @@ void CodeGen::genGenerateCode(void** codePtr, ULONG* nativeSizeOfCode)
 
     getEmitter()->emitBegFN(isFramePointerUsed()
 #if defined(DEBUG)
-                                ,
-                            (compiler->compCodeOpt() != Compiler::SMALL_CODE) &&
-                                !(compiler->opts.eeFlags & CORJIT_FLG_PREJIT)
+                            , (compiler->compCodeOpt() != Compiler::SMALL_CODE) &&
+                                !compiler->opts.jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT)
```

Comment:
`JitFlags` isn't a scoped `enum` so `JitFlags::` is not required. And if you make it a scoped enum then the JIT_FLAG_ prefix should go away.